inscriptions/ISic0246.xml

```diff
@@ -170,40 +175,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR127512" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="VM">
                 <ab>
-                    <lb n="1"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>ar
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="2"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">b</supplied>
-                    <expan>
-                        <abbr>en</abbr>
-                        <ex>e</ex>
-                    </expan>
-                    mer<unclear>e</unclear>n
-                    <unclear>t</unclear>
-                    <supplied reason="lost">i</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="3"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <unclear>v</unclear>
-                    <expan>
-                        <abbr>ixi</abbr>
-                        <ex>t</ex>
-                    </expan>
-                    <expan>
-                        <abbr>an</abbr>
-                        <ex>nis</ex>
-                    </expan>
-                    LXXX.
-                    <lb n="4"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>ar<gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">b</supplied><expan><abbr>en</abbr><ex>e</ex></expan> <g type="interpunct">·</g> mer<unclear>e</unclear>n<unclear>t</unclear><supplied reason="lost">i</supplied> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
`<gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">b</supplied><expan><abbr>en</abbr><ex>e</ex></expan>`
The supplied 'b' needs to be inside the <abbr> element

Reply:
So, it should be: <gap reason="lost" extent="unknown" unit="character"/><expan><abbr><supplied reason="lost">b</supplied>en</abbr><ex>e</ex></expan> ?

Reply:
Sorry... `<gap reason="lost" extent="unknown" unit="character"/><expan><abbr><supplied reason="lost">b</supplied>en</abbr><ex>e</ex></expan>`

Reply:
yes